apps/modernization-ui/src/SkipLink/SkipLink.scss

```diff
@@ -0,0 +1,6 @@
+.hidden-link {
+    color: transparent;
```

Comment:
issue:  The element is still visible as a giant bar on the top of the screen.  `display: none` might be enough

Reply:
I also deleted the styles as we don't need them